The dist tag is meant to be used for official builds and pulls certain
files, like assets or templates from the official branding. When doing
a development build unofficial ones should be used.

---
diff --git a/cmd/frontend/build.sh b/cmd/frontend/build.sh
index 75beee320..4af51f347 100755
--- a/cmd/frontend/build.sh
+++ b/cmd/frontend/build.sh
@@ -12,7 +12,7 @@ export CGO_ENABLED=0

 echo "--- go build"
 pkg="github.com/sourcegraph/sourcegraph/cmd/frontend"
-go build -trimpath -ldflags "-X github.com/sourcegraph/sourcegraph/internal/version.version=$VERSION  -X github.com/sourcegraph/sourcegraph/internal/version.timestamp=$(date +%s)" -buildmode exe -tags dist -o "$OUTPUT/usr/local/bin/$(basename $pkg)" "$pkg"
+go build -trimpath -ldflags "-X github.com/sourcegraph/sourcegraph/internal/version.version=$VERSION  -X github.com/sourcegraph/sourcegraph/internal/version.timestamp=$(date +%s)" -buildmode exe -o "$OUTPUT/usr/local/bin/$(basename $pkg)" "$pkg"

 echo "--- docker build $IMAGE"
 docker build -f cmd/frontend/Dockerfile -t "$IMAGE" "$OUTPUT" \
